lib/minify.js

```diff
@@ -36,6 +36,7 @@ function minify(files, options) {
         options = defaults(options, {
             compress: {},
             ie8: false,
+            safari10: true,
```

Comment:
Default should be `false`, just like `ie8`. These are special case options, so don't force it upon everyone.